one/src/main.rs

```diff
@@ -61,7 +61,7 @@ struct DaemonOpts {
     /// Listen address of the p2p swarm.
     #[arg(
         long,
-        default_values_t = vec!["/ip4/0.0.0.0/tcp/0".to_string(), "/ip4/0.0.0.0/udp/0/quic-v1".to_string()],
+        default_values_t = vec!["/ip4/0.0.0.0/tcp/4001".to_string(), "/ip4/0.0.0.0/udp/4001/quic-v1".to_string()],
```

Comment:
Can we run both tcp and quic on a single port?

Reply:
Yes they are TCP vs UDP and so can use the same port number.

Reply:
Thanks, I was suspecting this but wasn't sure!